snomed/com.b2international.snowowl.snomed.datastore.tests/src/com/b2international/snowowl/snomed/core/ecl/SnomedEclEvaluationRequestHistorySupplementTest.java

```diff
@@ -0,0 +1,186 @@
+				.parents(REFSET_HISTORICAL_ASSOCIATION_L)
+				.build());
+		
+		// generate a focus concept substance hierarchy
+		indexRevision(MAIN, DocumentBuilders.concept(SUBSTANCE)
+				.build());
+		
+		indexRevision(MAIN, DocumentBuilders.concept(SUBSTANCE_CHILD_CONCEPT)
+				.parents(SUBSTANCE_L)
+				.build());
+		
+		// generate association members
+		indexRevision(MAIN, SnomedRefSetMemberIndexEntry.builder()
+				.id(UUID.randomUUID().toString())
+				.active(true)
+				.moduleId(Concepts.MODULE_SCT_CORE)
+				.referencedComponentId(INACTIVE_CONCEPT_1)
+				.referenceSetType(SnomedRefSetType.ASSOCIATION)
+				.refsetId(Concepts.REFSET_SAME_AS_ASSOCIATION)
+				.targetComponentId(SUBSTANCE)
+				.build());
+		
+		indexRevision(MAIN, SnomedRefSetMemberIndexEntry.builder()
+				.id(UUID.randomUUID().toString())
+				.active(true)
+				.moduleId(Concepts.MODULE_SCT_CORE)
+				.referencedComponentId(INACTIVE_CONCEPT_2)
+				.referenceSetType(SnomedRefSetType.ASSOCIATION)
+				.refsetId(Concepts.REFSET_WAS_A_ASSOCIATION)
+				.targetComponentId(SUBSTANCE_CHILD_CONCEPT)
+				.build());
+		
+		indexRevision(MAIN, SnomedRefSetMemberIndexEntry.builder()
+				.id(UUID.randomUUID().toString())
+				.active(true)
+				.moduleId(Concepts.MODULE_SCT_CORE)
+				.referencedComponentId(INACTIVE_CONCEPT_3)
+				.referenceSetType(SnomedRefSetType.ASSOCIATION)
+				.refsetId(Concepts.REFSET_REPLACED_BY_ASSOCIATION)
+				.targetComponentId(SUBSTANCE)
+				.build());
+		
+		indexRevision(MAIN, SnomedRefSetMemberIndexEntry.builder()
+				.id(UUID.randomUUID().toString())
+				.active(true)
+				.moduleId(Concepts.MODULE_SCT_CORE)
+				.referencedComponentId(INACTIVE_CONCEPT_4)
+				.referenceSetType(SnomedRefSetType.ASSOCIATION)
+				.refsetId(Concepts.REFSET_PARTIALLY_EQUIVALENT_TO_ASSOCIATION)
+				.targetComponentId(SUBSTANCE_CHILD_CONCEPT)
+				.build());
+		
+		indexRevision(MAIN, SnomedRefSetMemberIndexEntry.builder()
+				.id(UUID.randomUUID().toString())
+				.active(true)
+				.moduleId(Concepts.MODULE_SCT_CORE)
+				.referencedComponentId(INACTIVE_CONCEPT_5)
+				.referenceSetType(SnomedRefSetType.ASSOCIATION)
+				.refsetId(Concepts.REFSET_POSSIBLY_EQUIVALENT_TO_ASSOCIATION)
+				.targetComponentId(SUBSTANCE)
+				.build());
+		
+		indexRevision(MAIN, SnomedRefSetMemberIndexEntry.builder()
+				.id(UUID.randomUUID().toString())
+				.active(true)
+				.moduleId(Concepts.MODULE_SCT_CORE)
+				.referencedComponentId(INACTIVE_CONCEPT_5)
+				.referenceSetType(SnomedRefSetType.SIMPLE)
+				.refsetId(SIMPLE_REFSET_ID)
+				.build());
+	}
+	
+	@Test
+	public void profile_default() throws Exception {
+		Expression actual = eval("<< " + SUBSTANCE + "  {{ + HISTORY }}"); // this equals with MAX profile
+		Expression expected = SnomedConceptDocument.Expressions.ids(Set.of(SUBSTANCE, SUBSTANCE_CHILD_CONCEPT, INACTIVE_CONCEPT_1, INACTIVE_CONCEPT_2, INACTIVE_CONCEPT_3, INACTIVE_CONCEPT_4, INACTIVE_CONCEPT_5));
+		assertEquals(expected, actual);
```

Comment:
(Optionally, add a message to these assertions telling the reader which historical association reference sets belong to the profile. They can be found in the specification and the `switch` expression that evaluates the query, but would be good to have it here as well, for reference.)

Reply:
Done.